FreshRSS/FreshRSS
# Added: Dark mode for Origine +Origine compact themes

Ref #4515
Ref #2793
Follow up of #4693

This PR shows how easy it is to use the variables to create cool new stuff.

This themes have now a dark mode:
- Origine
- Origine compact

![grafik](https://user-images.githubusercontent.com/1645099/201529360-03622ca1-8418-4c5a-8c78-dda1af7fb668.png)


Changes proposed in this pull request:

- media query that ask for the browsers preferences (CSS)


How to test the feature manually:

1. if you have a dark mode in your browser enabled: you see the dark theme
2. if you have NOT a dark mode in your browser enabled: you see the (default) light theme


Pull request checklist:
